tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts

```diff
@@ -167,3 +164,63 @@ test('Verify that user can see that Command Helper displays the previous informa
     //Verify Command helper information
     await t.expect(cliPage.cliHelperTitleArgs.textContent).contains(commandForCheck, 'Command Helper information persists after reopening');
 });
+test.only('Verify that user can work with Gears group in Command Helper (RedisGears module)', async t => {
+    const commandForCheck = 'RG.GETEXECUTION';
+    const commandArgumentsCheck = 'RG.GETEXECUTION id [SHARD|CLUSTER]';
+    const readMoreLink = '#rggetexecution';
```

Comment:
1. I'm not saying it's bad, but you created here variables while you use with only one time within the test.
2. The two first are already defined on top, so you could use with `let` in order to re-use here.

Reply:
Thanks! I re-worked all tests from this file